packages/cli/src/utils/plugins.ts

```diff
@@ -276,20 +281,74 @@ const addPluginsTheme = async (basePath: string, plugins: Plugin[]) => {
         getPackagePath(getPluginName(plugin), 'src', 'themes', 'index.scss')
       )
     )
-    .map(
-      (plugin) => `@import "${getPluginName(plugin)}/src/themes/index.scss";`
-    )
+    .map((plugin) => `@import "${getPluginName(plugin)}/src/themes/index.scss"`)
     .join('\n')
 
   writeFileSync(tmpThemesPluginsFile, pluginImportsContent)
 }
 
+const getPluginAPIFileContent = (pluginName: string, apiName: string) => `
+// GENERATED FILE
+// @ts-nocheck
+import apiHandle from 'src/plugins/${pluginName}/apis/${apiName}'
+import { NextApiRequest, NextApiResponse } from "next/types"
+
+export default function handle(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
I'd say that we need to not pass down a NextRequest, because we don't want to expose anything "Next" to the plugin authors. Can we convert this to a [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request)/[Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) from the Web API?

Reply:
You're correct!